test/tools/libtesteth/ImportTest.cpp

```diff
@@ -153,7 +153,7 @@ bytes ImportTest::executeTest()
 							}
 
 							json_spirit::mObject expetSectionObj;
-							expetSectionObj["network"] = exp.get_obj().at("network");
```

Comment:
what was the issue again?  hive tests were working before that.

Reply:
this is a crucial change. we have to check that everything is correct

Reply:
I mean double check

Reply:
The issue was this
```
                "network" : [
                    "ALL"
                ],
                "result" : {
                    "0x095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
                        "balance" : "0x0de0b6b3a76586a0",
                        "code" : "0x6001600101600055",
                        "storage" : {
                            "0x00" : "0x02"
                        }
                    },
                    "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
                        "balance" : "0x4563918244f4a034",
                        "code" : "",
                        "nonce" : "0x00",
                        "storage" : {
                        }
                    },
                    "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
                        "balance" : "0x0de0b6b3a761d92c",
                        "code" : "",
                        "nonce" : "0x01",
                        "storage" : {
                        }
                    }
                }
            },
            {
                "network" : [
                    "ALL"
                ],
                "result" : {
                    "0x095e7baea6a6c7c4c2dfeb977efac326af552d87" : {
                        "balance" : "0x0de0b6b3a76586a0",
                        "code" : "0x6001600101600055",
                        "storage" : {
                            "0x00" : "0x02"
                        }
                    },
                    "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba" : {
                        "balance" : "0x4563918244f4a034",
                        "code" : "",
                        "nonce" : "0x00",
                        "storage" : {
                        }
                    },
                    "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b" : {
                        "balance" : "0x0de0b6b3a761d92c",
                        "code" : "",
                        "nonce" : "0x01",
                        "storage" : {
                        }
                    }
                }
            },
```

Reply:
^ you see `"network" : [ "ALL" ] ` is repeated?